x-pack/plugins/canvas/public/components/workpad_page/workpad_interactive_page/workpad_interactive_page.scss

```diff
@@ -1,20 +1,9 @@
-.canvasPage,
-.canvasPage:focus {
+.canvasInteractivePage {
   z-index: initial;
   box-shadow: 0 0 7px 0 rgba(0, 0, 0, 0.2);
```

Comment:
`@include euiBottomShadowFlat(opacity: .4)` will make this match EUI shadows and theming (ours is a darker blue).

Reply:
Done, thanks!